app/src/main/java/com/raghav/spacedawn/ui/MainActivity.kt

```diff
@@ -42,29 +44,25 @@ class MainActivity : AppCompatActivity() {
                             )
                         )
                     } catch (e: ActivityNotFoundException) {
-                        startActivity(
-                            Intent(
-                                ACTION_VIEW,
-                                Uri.parse("http://play.google.com/store/apps/details?id=$packageName")
-                            )
+                        customTabIntent.launchUrl(
+                            this,
+                            Uri.parse("http://play.google.com/store/apps/details?id=$packageName")
```

Comment:
in this case, we can rely on ACTION_VIEW intents here to launch the play store before this ?

Reply:
I read this from the documentation hence didn't added any check
https://developer.chrome.com/docs/android/custom-tabs/integration-guide/#what-happens-if-the-user-doesnt-have-a-browser-that-supports-custom-tabs-installed
<img width="812" alt="Screenshot 2023-02-27 at 7 54 34 PM" src="https://user-images.githubusercontent.com/122787801/221589163-18a2c15d-1542-4e94-9826-3e85739481b0.png">